### ISSUE TITLE
Buttons: Update CSS styles for buttons with disabled state

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#16452]
In WordPress 5.5, the styles for both primary and secondary buttons were updated in the WordPress admin.

![image](https://user-images.githubusercontent.com/426388/87149522-b41c2f00-c2b0-11ea-9cd6-eb00b1ce4230.png)

We may need to update some of our buttons' designs to match:

**In the Jetpack dashboard**

![image](https://user-images.githubusercontent.com/426388/87149569-c8f8c280-c2b0-11ea-8b79-0dea602db1d7.png)
![image](https://user-images.githubusercontent.com/426388/87149629-e9288180-c2b0-11ea-986a-dc9ff36435f4.png)

**In JITM**

![image](https://user-images.githubusercontent.com/426388/87149938-710e8b80-c2b1-11ea-8025-2042da17c317.png)

********

Reference:
https://make.wordpress.org/core/2020/07/10/new-css-styles-for-buttons-with-disabled-state-in-wp-5-5/

Primary issue: #15388

**Comment by folletto** [comment]
What design is needed? I think the change is quite straightforward: we just need to use the new style.
Or... am I missing something?

**Comment by jeherve** [comment]
> I think the change is quite straightforward: we just need to use the new style.

Yes, I'd think so too. I assume it's fine to have that new style even on sites that still run WordPress 5.4.

**Comment by folletto** [comment]
Yup — ok thanks for clarifying.

We might want to highlight if we spot any button that clashes with out UI, but I'd review that on a case-by-case basis.

**Comment by jeherve** [comment]
Also in #21734, mentioning the accessibility issues of the old buttons.

**Comment by Robertght** [comment]
This is what I see on my end:

![Image](https://github.com/user-attachments/assets/4450d3c4-aaa0-4090-a28f-8fed6591150e)


I assume this is still an issue so marking it as Triaged.

**Comment by chrisbliss18** [comment]
In 2023, a design refresh was done (p1HpG7-lJK-p2). The Figma design included new styling for the buttons, including deactivated states. Work in [this PR](https://github.com/Automattic/jetpack/pull/30166) applied those designs.

I reviewed through the Jetpack pages and found that the button styling is consistent with the guide. I also did not find any issues with the JITM buttons.

I'm closing this issue due to it no longer being relevant.
